apps/meteor/client/views/admin/rooms/RoomsTableFilters.tsx

```diff
@@ -27,46 +22,52 @@ const roomTypeFilterStructure = [
 		checked: false,
 	},
 	{
-		id: 'omnichannel',
+		id: 'l',
 		text: 'Omnichannel',
 		checked: false,
 	},
 	{
-		id: 'teams',
-		text: 'Teams',
+		id: 'p',
+		text: 'Private_Channels',
 		checked: false,
 	},
-] as OptionProp[];
-
-const roomVisibilityFilterStructure = [
 	{
-		id: 'filter_by_visibility',
-		text: 'Filter_by_visibility',
-		isGroupTitle: true,
-	},
-	{
-		id: 'private',
-		text: 'Private',
+		id: 'c',
+		text: 'Public_Channels',
 		checked: false,
 	},
 	{
-		id: 'public',
-		text: 'Public',
+		id: 'teams',
+		text: 'Teams',
 		checked: false,
 	},
 ] as OptionProp[];
 
+// TODO: Revisit the back-end of the rooms page in order for this filter to be possible
+// const roomVisibilityFilterStructure = [
+// 	{
+// 		id: 'filter_by_visibility',
+// 		text: 'Filter_by_visibility',
+// 		isGroupTitle: true,
+// 	},
+// 	// {
+// 	// 	id: 'public',
+// 	// 	text: 'Public',
+// 	// 	checked: false,
+// 	// },
+// ] as OptionProp[];
+
 const RoomsTableFilters = ({ setFilters }: { setFilters: Dispatch<SetStateAction<any>> }): ReactElement => {
 	const t = useTranslation();
 	const [text, setText] = useState('');
 	const [roomTypeOptions, setRoomTypeOptions] = useState<OptionProp[]>(roomTypeFilterStructure);
-	const [roomVisibilityOptions, setRoomVisibilityOptions] = useState<OptionProp[]>(roomVisibilityFilterStructure);
+	// const [roomVisibilityOptions, setRoomVisibilityOptions] = useState<OptionProp[]>(roomVisibilityFilterStructure);
```

Comment:
why leave it commented?

Reply:
We are going to implement when the backend does that

Reply:
I would recommend not leaving comments on the code and reimplement when backend is done